### PR TITLE
VY-RES-181 Create table structure for project history

### PR DESCRIPTION
**Pull request by krishnatimilsina**
@lf-kailashraj @anishmanandhar 

Please review these codes.

### CODE REVIEW

File: resources/database/vyaguta-resource.sql [hunk 1]
```diff
@@ -133,6 +133,22 @@ ALTER TABLE ONLY project_members
 ALTER TABLE ONLY project_members 
 	ADD CONSTRAINT projects_fk FOREIGN KEY(project_id) REFERENCES projects ON DELETE CASCADE;
 	
+CREATE TABLE project_histories (
+    id character varying(32) NOT NULL,
+    project_id character varying(32) NOT NULL,
+    batch_id character varying(32) NOT NULL,
```
Comment (lf-kailashraj) [inline resources/database/vyaguta-resource.sql]: lets make it batch_no

File: resources/database/vyaguta-resource.sql [hunk 2]
```diff
@@ -133,6 +133,22 @@ ALTER TABLE ONLY project_members
...
+    attribute character varying(255) NOT NULL,
+    old_value character varying(255),
+    new_value character varying(255),
+    reason text,
+    created_by character varying(32) NOT NULL,
+    updated_by character varying(32),
```
Comment (lf-kailashraj) [inline resources/database/vyaguta-resource.sql]: updated_by and updated_at isn't necessary. data in this table isn't going to be updated.

Comment (lf-kailashraj) [inline resources/database/vyaguta-resource.sql]: same case in project member history as well

Comment (krishnatimilsina) [inline resources/database/vyaguta-resource.sql]: I have updated on VY-RES-182.

**Pull request merged**